wherehows-web/app/routes/index.js

```diff
@@ -1,7 +1,22 @@
 import Ember from 'ember';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import fetch from 'ember-network/fetch';
```

Comment:
fyi, looks like `ember-fetch` is replacing `ember-network` .
Based on conversations in https://github.com/samselikoff/ember-cli-mirage/issues/983, https://github.com/tomdale/ember-network/issues/17 and mostly https://github.com/stefanpenner/ember-fetch/issues/21